rust/main/agents/relayer/src/msg/op_batch.rs

```diff
@@ -51,7 +51,7 @@ impl OperationBatch {
 
             if mailbox.supports_batching() {
                 warn!(excluded_ops=?excluded_ops, "Either operations reverted in the batch or the txid wasn't included. Sending them back to prepare queue.");
-                let reason = ReprepareReason::ErrorEstimatingGas;
```

Comment:
this should stay as is, otherwise we get high urgency errors. `ErrorSubmitting` should be set only when simulation was successful but submission still failed